### DISCUSSION TITLE
Sometimes loading of HFS gives an error in the console

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#308] [General]
Hi Rejetto,

Sometimes the loading of HFS gives in error in the console. I think this happens always when I start HFS for the first time after starting my computer. If I close HFS en start HFS again then then the error is gone. A normal startup of HFS.

If I have the error in the console then there are also no files to see in my FTP server. I hope that this makes it clear to you what I mean. See my screenshot. The error is present in it. It can not find the config file in the system32 directory.

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl

![image](https://github.com/rejetto/hfs/assets/140949730/b8f695c4-7501-46dd-b155-7b23eac577ca)

**Comment by rejetto** [comment]
are you keeping HFS in \windows\system32 ?

**Reply by PH5HP** [reply]
Hi Rejetto,

No. Not at all. HFS is in a sub directory of Communication. The path is c:/Communication/HFS Beta

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl

**Comment by rejetto** [comment]
so, what's wrong there is that HFS is running in the wrong folder
<img width="191" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/1a21e204-7ce5-4d07-8d68-51c7c90ffc82">

cwd is "current working directory". Is a concept of the operating system, it is where the data should be, basically.
Clearly that's the wrong folder. It's something to do with the way you are running it. Are you maybe running it as a service?

**Reply by PH5HP** [reply]
Hi Rejetto,

No, I am running it as a normal program. The only thing I can think of is that the auto running of HFS when my Windows 10 starts causes the problem. Maube it runs before Windows 10 starting is 100% done. If I start HFS again manually nothing is wrong. The second time I do not get any error messages at all.

Here is a screenshot of the second time I started HFS:

![image](https://github.com/rejetto/hfs/assets/140949730/f0dba7e2-a08c-41e3-93ca-a0324ca732cb)

**Comment by rejetto** [comment]
i don't think the point is the "100% done".
let's have a closer look at the details of your "hfs in startup".
it is possibly a "shortcut" that has properties.

**Reply by PH5HP** [reply]
Hi Rejetto,

Maybe because I started HFS-beta with not a shortcut automaticly, but directly the exe-file. I changed it. Now it will startup automaticly by loading it with a shortcut.

Let see what happens the next time I startup my computer. I will let you know.

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl

**Comment by SanokKule** [comment]
Did you by any chance create a Task Scheduler task?
I tried to do this without specifying a start directory and sure enough it started in system32...
![image](https://github.com/rejetto/hfs/assets/24497821/36ab5f86-ae5a-4da0-ae61-2467bf301867)

**Reply by SanokKule** [reply]
If you want it to start properly, also specify a starting directory like so:
![image](https://github.com/rejetto/hfs/assets/24497821/027ba024-9aef-4369-b536-726ddb639924)

**Reply by PH5HP** [reply]
Hi Rejetto,

I wiil try this.

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl

**Reply by SanokKule** [reply]
I'm not [Rejetto](https://github.com/rejetto) 😅.
You're welcome though, i had that happen before with some other program.
